cookbook/tools/telegram_tools.py

```diff
@@ -0,0 +1,13 @@
+from phi.agent import Agent
+from phi.tools.telegram import TelegramTools
+
+
```

Comment:
Please add instructions here on how to signup, then get the telegram token and chat ID

Reply:
Thank you for the suggestion! However, users of this tool are generally expected to know how to obtain their Telegram token and chat ID, just as we didn’t include instructions for obtaining a Slack token. That said, if there’s a specific reason to provide additional guidance for Telegram, I’d be happy to revisit this. Please let me know your thoughts!

Reply:
We are just trying to move into a more explicit direction, giving users as much help as possible. I don't think users of telegram automatically know how to get these "developer" kind of credentials.

I use telegram and I didn't know how to get these.

Reply:
Thank you for the feedback! I’ve added comments explaining how to obtain the token and chat ID. Could you please review them and let me know if there’s anything else needed?